### PR TITLE
Move JRC_IDEES from mrremind to mrcommons.

### PR DESCRIPTION
**Pull request by robinhasse**
I want to make use of the IDEES data base in `mredgebuildings` as well an therefore move it from `mrremind` to `mrcommons`.

See [#284](https://github.com/pik-piam/mrremind/pull/284) in `mrremind`.

### CODE REVIEW

File: man/calcGDPpppPast.Rd [hunk 1]
```diff
@@ -15,8 +15,8 @@ GDP PPP(ICP11) in million USD05 equivalents
 \description{
 \ifelse{html}{\href{https://lifecycle.r-lib.org/articles/stages.html#deprecated}{\figure{lifecycle-deprecated.svg}{options: alt='[Deprecated]'}}}{\strong{[Deprecated]}}
 
-This function is defunct. Use \code{\link[mrdrivers:calcGDP]{mrdrivers::calcGDPPast()}} instead.
```
Comment (LaviniaBaumstark) [inline man/calcGDPpppPast.Rd:18]: I thought that we do not have calcGDPpast any more and should use calcGDP. Why did you change that?

Comment (robinhasse) [reply man/calcGDPpppPast.Rd]: You're right. I did not changes that intentionally but had outdated packages. Changes to most `.Rd` files vanished after updates.
